pkgs/development/python-modules/miniflux/default.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  requests,
+  fetchPypi,
+}:
+buildPythonPackage rec {
+  pname = "miniflux";
+  version = "1.1.4";
+  src = fetchPypi {
+    pname = "miniflux";
+    inherit version;
+    hash = "sha256-VFcddb3rU6NHF2aRuWFNpqLrtI94fChQ+DWhh0WHhN4=";
+  };
+  pythonImportsCheck = [ "miniflux" ];
+  pyproject = true;
+  build-system = [ setuptools ];
+  dependencies = [ requests ];
+  meta = {
+    description = "Miniflux Python API Client";
+    homepage = "https://github.com/miniflux/python-client";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ wariuccio ];
+  };
+}
```

```suggestion
{
  lib,
  buildPythonPackage,
  fetchFromGitHub,
  setuptools,
  requests,
  pytestCheckHook,
}:

buildPythonPackage rec {
  pname = "miniflux";
  version = "1.1.4";
  pyproject = true;
  
  src = fetchFromGitHub {
    owner = "miniflux";
    repo = "python-client";
    tag = version;
    hash = "";
  };

  build-system = [ setuptools ];
  
  dependencies = [ requests ];
  
  pythonImportsCheck = [ "miniflux" ];
  
  nativeCheckInputs = [ pytestCheckHook ];
  
  meta = {
    description = "Miniflux Python API Client";
    homepage = "https://github.com/miniflux/python-client";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ wariuccio ];
  };
}
```

nice work. just needs to be formatted a bit + fetch from github/source directly instead of pypi. 

also enable pytest checks/tests.

---

Nevermind, it did run some tests :)